homeassistant/components/unifi_access/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for the UniFi Access integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import uiaccessclient
+import urllib3.exceptions
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_API_TOKEN, CONF_HOST, CONF_VERIFY_SSL
+from homeassistant.core import HomeAssistant
+
+from .const import DEFAULT_HOST, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=DEFAULT_HOST): str,
```

Comment:
So users host this themselves? What should the user enter? An url with port?

Because
- In that URL in `__init__.py` we assume that they have this hosting on `https`, is that required for this kind of software?
- Can they host it on different ports?

Reply:
Little quick on the last one and forgot to answer as to the port, but as far as I can see that port is also fixed. I don't see any exposed configuration on my UniFi console to change it, and the documentation for the API seems to indicate that both HTTPS and port is fixed as is.